syntax = "proto3";
package rofi.messages;

message RofiDescription {
    int32 rofiId = 1;

    int32 jointCount = 2;
    int32 connectorCount = 3;
}
